### ISSUE TITLE
wrong output path with report file in dev mode

### ISSUE THREAD

**Issue by ikobe** [th0r/webpack-bundle-analyzer#75]
#### Issue description

with this [commit](https://github.com/th0r/webpack-bundle-analyzer/commit/431ebbc61540a2950f7eb1084b8012ba48ce2bbf), i got wrong filepath with the report file in dev mode

#### Technical info
* Webpack Bundle Analyzer version: 2.8.1
* Webpack version: 1.12
* Node.js version:7.6.0
* npm/yarn version:  4.1.2
* OS: mac os

#### Debug info

with webpack-dev-server

```js
new BundleAnalyzerPlugin({
    analyzerMode: 'static',
    reportFilename: 'report.html',
    openAnalyzer: false,
    generateStatsFile: true,
    statsFilename: 'stats.json',
    statsOptions: null,
    source: false,
    logLevel: 'silent'
}),
```

**Comment by ikobe** [comment]
@valscion will you  please fix this bug quickly ? it has affects many projects of mine

❤️

**Comment by valscion** [comment]
To work around this, feel free to use an older version of `webpack-bundle-analyzer`.

```js
npm install --save-dev webpack-bundle-analyzer@2.7.0
```

To fix this bug sooner rather than later, it would be great to have a reproduction repository that shows this issue.

**Comment by th0r** [comment]
Fixed in `v2.8.2`
